<launch>

	<arg name="simulation" value="true"/>

	<arg name="mujoco_cam_file" value="$(find estimation)/config/mujoco_cam.yaml"/>
	<arg name="logitech_cam_file" value="$(find estimation)/config/cTw_cam0.yaml"/>

	<arg name="cam_file" if="$(arg simulation)" value="$(arg mujoco_cam_file)" />
  <arg name="cam_file" unless="$(arg simulation)" value="$(arg logitech_cam_file)" />
  
  <arg name="current_ns" value="mushr"/>

  <param name="node_name" value="aruco_wTc"/>
  <group ns="$(arg current_ns)">
 
    <node pkg="estimation" type="nodelets_as_nodes" name="estimation_nodelets_as_nodes" output="screen">
      <rosparam command="load" file="$(arg cam_file)" />
      <param name="pose_topic" value="/$(arg current_ns)/aruco_cTw/robot_pose" />
      <param name="image_topic" value="/$(arg current_ns)/camera/rgb"/>
      <param name="goal_pose_topic" value="/$(arg current_ns)/goal_pos"/>
			<param name="goal_rad_topic" value="/$(arg current_ns)/goal_radius"/>
			<param name="trajectory_topic" value="/$(arg current_ns)/ml4kp_traj"/>
			<param name="camera_frame" value="camera"/>
			<param name="world_frame" value="world"/>
			<param name="robot_frame" value="robot_0"/>
    </node>   

   </group>
</launch>
